packages/utils/src/index.ts

```diff
@@ -0,0 +1,299 @@
+import * as hpke from '@hpke/core';
+import { subtle, webcrypto} from 'crypto';
+
+
+// Exported Utilities
```

Comment:
i think we can split this file into a couple different ones to separate concerns. maybe based on: 

- key material (generating a target key, importing a credential, compressing/uncompressing, decrypt)
- math
- encoding/decoding

Reply:
Good call - Totaly agreed!